Cannot build on CI environment

### Issue details
The file [mavros_msgs/mavlink_convert.h](https://github.com/mavlink/mavros/blob/master/mavros_msgs/include/mavros_msgs/mavlink_convert.h) depends on `#include <mavconn/mavlink_dialect.h>`.
However, this file is not produced in a CI environment when we build `libmavconn`, as we can see from [libmavconn/CMakeLists.txt](https://github.com/mavlink/mavros/blob/master/libmavconn/CMakeLists.txt):

![image](https://user-images.githubusercontent.com/51488981/85064774-aec83a80-b171-11ea-89f8-446d5a89f996.png)

### Possible Solution
- Remove the conditions related to CI environment

### MAVROS version and platform

Mavros: 1.2.0
ROS: Kinetic
Ubuntu: 16.04 / CI environment

### Node logs

```
Errors     << mavros:make /root/rosws/logs/mavros/build.make.000.log
In file included from /root/rosws/src/mavros/mavros/include/mavros/utils.h:22:0,

                 from /root/rosws/src/mavros/mavros/src/lib/enum_to_string.cpp:19:

/root/rosws/src/mavros/mavros_msgs/include/mavros_msgs/mavlink_convert.h:18:37: fatal error: mavconn/mavlink_dialect.h: No such file or directory

compilation terminated.

make[2]: *** [CMakeFiles/mavros.dir/src/lib/enum_to_string.cpp.o] Error 1

make[2]: *** Waiting for unfinished jobs....

In file included from /root/rosws/src/mavros/mavros/include/mavros/utils.h:22:0,

                 from /root/rosws/src/mavros/mavros/src/lib/enum_sensor_orientation.cpp:17:

/root/rosws/src/mavros/mavros_msgs/include/mavros_msgs/mavlink_convert.h:18:37: fatal error: mavconn/mavlink_dialect.h: No such file or directory

compilation terminated.

make[2]: *** [CMakeFiles/mavros.dir/src/lib/enum_sensor_orientation.cpp.o] Error 1

make[1]: *** [CMakeFiles/mavros.dir/all] Error 2

make: *** [all] Error 2

cd /root/rosws/build/mavros; catkin build --get-env mavros | catkin env -si  /usr/bin/make --jobserver-fds=6,7 -j; cd -


Failed     << mavros:make                                  [ Exited with code 2 ]
Failed    <<< mavros                                       [ 4.9 seconds ]
```

--- a/libmavconn/CMakeLists.txt
+++ b/libmavconn/CMakeLists.txt
@@ -65,15 +65,11 @@ em_expand(${CMAKE_CURRENT_SOURCE_DIR}/mavlink.context.py.in
   ${CMAKE_CURRENT_SOURCE_DIR}/src/mavlink_helpers.cpp.em
   ${CMAKE_CURRENT_BINARY_DIR}/catkin_generated/src/mavlink_helpers.cpp)
 
-if(DEFINED ENV{CI})
-  message(STATUS "CI environment detected. Assume that it is on RO fs")
-else()
-  message(STATUS "Copy mavlink_dialect.h to source tree")
-  file(
-    COPY ${CMAKE_CURRENT_BINARY_DIR}/catkin_generated/include/mavconn/mavlink_dialect.h
-    DESTINATION ${CMAKE_CURRENT_SOURCE_DIR}/include/mavconn/
-    )
-endif()
+message(STATUS "Copy mavlink_dialect.h to source tree")
+file(
+  COPY ${CMAKE_CURRENT_BINARY_DIR}/catkin_generated/include/mavconn/mavlink_dialect.h
+  DESTINATION ${CMAKE_CURRENT_SOURCE_DIR}/include/mavconn/
+  )
 
 #############
 ## Install ##

